tools/packaging/static-build/qemu.blacklist

```diff
@@ -5,7 +5,7 @@
 qemu_black_list=(
 */bin/qemu-pr-helper
 */bin/virtfs-proxy-helper
-*/libexec/
+*/libexec/kata-qemu/qemu*
```

Comment:
Hmm. What's the reason of this?

Reply:
We are skipping these qemu helpers under `/libexec` but `virtiofsd` was installed there as well. So I modified it to just skip qemu- prefixing binaries to exclude virtiofsd.